src/main/resources/schema.sql

```diff
@@ -0,0 +1,99 @@
+
+create table binary_content
```

Comment:
create table과 같은 SQL 예약어는 대문자로 쓰면 가독성이 더 좋아질 것 같아요.  
```suggestion
CREATE TABLE binary_content
```

참고: [SQL - 가독성 좋은 SQL 작성하기](https://green-bin.tistory.com/124)